src/index.ts

```diff
@@ -22,10 +22,12 @@ import gfm from 'remark-gfm';
  *
  * @param body any Markdown or GFM content
  */
-export function markdownToBlocks(body: string): notion.Block[] {
+export function markdownToBlocks(
+  body: string,
+  unsupported = false
```

Comment:
```suggestion
  allowUnsupportedObjectType = false
```

Can we change this name for clarity